src/lib/pages/backoffice/DocumentRequest/DocumentRequestDetails/DocumentRequestSteps/DocumentRequestStepPanels/ChooseProviderStepPanel/ChooseProviderStepPanel.js

```diff
@@ -31,6 +31,8 @@ class AcqProvider extends Component {
           {
             document_pid: _get(docReq, 'document_pid', ''),
             patron_pid: docReq.patron_pid,
+            payment_mode: docReq.payment_method,
```

Comment:
If `payment_mode` is not required, thus ensured that is always there, I would go for something like 
```javascript
{
...,
payment_mode: _get(docReq, "payment_mode", <default_value_for_field_type>),
...
}
```